target/linux/ath79/image/Makefile

```diff
@@ -66,6 +74,28 @@ define Device/Default
 	append-rootfs | pad-rootfs | append-metadata | check-size $$$$(IMAGE_SIZE)
 endef
 
+define Device/seama
+$(Device/Default)
```

Comment:
`$(Device/Default)`is always included and can be dropped here.

Reply:
Will delete this line.